api/app/signals/apps/api/templates/api/pdf/print_signal.html

```diff
@@ -185,8 +185,8 @@ <h2>Foto's</h2>
             <p><img src="{{ image.file.url }}" style="width:680px" alt=""></p>
```

Comment:
Do we still need the `{% if images %}` part of the template? If it is not used I think it is better to remove it.

Reply:
I removed this